src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -23,20 +23,21 @@ export class RealLinesOfCode implements Metric {
      * @param allNodeTypes List of all configured syntax node types.
      */
     constructor(allNodeTypes: ExpressionMetricMapping[]) {
+        // Should I use this? What about the category == "comment" thing?
+        // getQueryStatements(allNodeTypes, this.getName());
```

Comment:
Please fix this unresolved todo

Reply:
I have removed that, as it is resolved in #32